go/cloudbuild-test.yaml

```diff
@@ -50,3 +50,20 @@ steps:
         "-v",
       ]
     waitFor: ["go123-build"]
+
+  # Go 1.23 release build
+  - name: gcr.io/cloud-builders/docker
+    args: ["build", "-t", "gcr.io/$PROJECT_ID/go123-release", "."]
```

Comment:
I think this and below should mirror the values in `cloudbuild-release.yaml` for consistency.

Reply:
Updated, PTAL.